### ISSUE TITLE
Rounded Corners on IOS not working

### ISSUE THREAD

**Issue by poochie89** [robinmanuelthiel/flexbutton#94]
When creating a togglebutton with rounded corners, it has no effect on iOS devices

**Comment by robinmanuelthiel** [comment]
Does the Demo Project in this repository work for you? It has rounded corners for toggle buttons...

**Comment by poochie89** [comment]
The Demo Project does work. I am 0.14.
I am Initializing FlexButton in AppDelegate so im quite puzzled. I copied the Togglebutton from the demo project, but in my project the rounded corners dont work

**Comment by poochie89** [comment]
Any updates or ideas on this?

**Comment by robinmanuelthiel** [comment]
You are right, the demo project does not work when using the NuGet package. Weird...
I was able to reproduce and will investigate.

**Comment by robinmanuelthiel** [comment]
Version 0.14.1 is out and should address this issue. @poochie89, can you please check, if this solves this for you? https://www.nuget.org/packages/Forms.Controls.FlexButton/0.14.1
